### DISCUSSION TITLE
[Feature], option to disable image, or any kind of media?

### DISCUSSION THREAD

**Discussion by johnsmithodom** [ArchiveBox/ArchiveBox#1234] [Q&A] [answered]
Let's say we would like to save a page, but only interested in text content.  Does archivebox have the option to choose not to save any kind of media, like image, mp3, video?  Thank you.

**Comment by pirate** [comment, marked as answer]
Yes, turn off everything except singlefile, readability, and mercury here: https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#archive-method-toggles

Also set https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#save_wget_requisites to False

**Reply by gerroon** [reply]
Hi I just want to make sure, SAVE_MEDIA=False will only disable videos, not images because I definitely want to keep the images.

**Reply by pirate** [reply]
correct, SAVE_MEDIA=False is not a very clear name, it only turns off videos, audio, and subtitles.
